pkg/limayaml/default.yaml

```diff
@@ -27,11 +27,15 @@ images:
 - location: "https://cloud-images.ubuntu.com/impish/current/impish-server-cloudimg-arm64.img"
   arch: "aarch64"
 
-# Specify QEMU CPU emulation, e.g., "host", "qemu64", "Haswell".
+# Specify QEMU CPU type for each arch. Only used for emulation; cpu type is always "host"
+# when the desired arch matches the host arch.
 # You can see what options are available for host emulation with: `qemu-system-$(arch) -cpu help`.
 # Setting of instructions is supported like this: "qemu64,+ssse3".
-# Default: "host" (native arch), "qemu64" (Intel on ARM), or "cortex-a72" (ARM on Intel)
-cpuType: null
+cpuType:
+  # Default: "cortex-a72"
+  aarch64: null
+  # Default: "qemu64"
+  x86_64: null
```

Comment:
Not new in this PR, but I feel `cpuType` should be explained in the "FURTHER ADVANCED" section

Reply:
I agree! That's why I asked for it in https://github.com/lima-vm/lima/pull/643#discussion_r805525700.

Unfortunately I don't have the answers either; otherwise I would add something.

Reply:
Oh, do you just mean it should be moved down in the file? I'm going to do that!

Reply:
Done